pom.xml

```diff
@@ -65,6 +65,16 @@
             <artifactId>gson</artifactId>
             <version>2.8.5</version>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.retrofit2</groupId>
+            <artifactId>retrofit</artifactId>
```

Comment:
These changes were mentioned in a separate code review so we will discount them.